frontend/src/container/NotificationWall.jsx

```diff
@@ -147,15 +147,15 @@ export class NotificationWall extends React.Component {
             return {
               icon: 'far fa-comments',
               title: props.t('Comment_noun'),
-              text: props.t('{{author}} commented on {{content}} in {{space}}', i18nOpts),
+              text: props.t('{{author}} commented on {{content}}', i18nOpts),
```

Comment:
Space and the span tags in the i18nOpts const (line 121), are still needed?

Reply:
No, I will delete it. Thanks :)